Redo account request pages (#52)
osg-htc/user-documentation: documentation/overview/account_setup/generate-add-sshkey.md

![alt text](https://raw.githubusercontent.com/OSGConnect/connectbook/master/images/puttygen_ssh_key.png "PuttyGen Window")

## Next Steps

After generating the key, you will need to upload it to a web profile. 

> * If your account has been created using XX, follow the instructions here: [](../ap7-access.md)

Reviewer: Incomplete

> * If your account has been created using XX, follow the instructions here: [](../ap7-access.md)

Reviewer 2: ```suggestion
* If your account was created through "OSG Manage", follow the instructions here: [](../ap7-access.md)
```

> * If your account has been created using YY, follow the instructions here: [](../connect-access.md)

Reviewer: Incomplete

> * If your account has been created using YY, follow the instructions here: [](../connect-access.md)

Reviewer 2: ```suggestion
* If your account was created through "OSG Connect" follow the instructions here: [](../connect-access.md)
```
